movies/movies/TableView/AboutMoviesPresenter.swift

```diff
@@ -0,0 +1,69 @@
+// AboutMoviesPresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+import UIKit
+
+//MARK: - Protocol
+
+protocol AboutMoviesPresenterProtocol: AnyObject {
+    var aboutMovie: Result? { get set }
+    init(view: AboutMoviesViewProtocol, networkService: NetworkServiceProtocol)
+    func fetchDataAboutMovie(cell: AboutMovieTableViewCell, indexPath: IndexPath, idMovie: Int)
+    func getImage(cell: AboutMovieTableViewCell)
+}
+
+protocol AboutMoviesViewProtocol: AnyObject {
+    func reloadData()
+}
+
+final class AboutMoviesPresenter: AboutMoviesPresenterProtocol {
+    
+    //MARK: - Public Properties
+    
+    var networkService: NetworkServiceProtocol!
```

Comment:
```suggestion
  private  var movieAPIService: NetworkServiceProtocol!
```
в тз написано так переименовать movieAPIService